platform/wayland/cog-platform-wl.c

```diff
@@ -350,11 +350,11 @@ pointer_on_button(void              *data,
             cog_popup_menu_handle_event(
                 popup->popup_menu, !!state ? COG_POPUP_MENU_EVENT_STATE_PRESSED : COG_POPUP_MENU_EVENT_STATE_RELEASED,
                 event.x, event.y);
-            cog_wl_platform_popup_update(platform);
+            cog_wl_platform_popup_update();
```

Comment:
The platform is a singleton, so it makes no sense to pass it as parameter. Just get it with `cog_platform_get()` as here: https://github.com/Igalia/cog/pull/667/files#diff-7342248ee4ad822cd294d9c8341a52fd10b3d1df4dd2a202ccb926df67d51034R1378